concepts/pipelines.md

```diff
@@ -1,5 +1,18 @@
 # Pipelines
 
-A pipeline is a container that consists of many elements and links between them. Like an Element, every Pipeline also has a playback state and on its basis, it manages the state of the contained elements.
+Conceptually, a pipeline is a container that consists of many elements and links between them.
+Like an element, every pipeline has a playback state (and uses the same callbacks to handle transitions between them).
+It's an Erlang process responsible for starting elements (usually referred to as children) and
```

Comment:
- Start with technically/physically would be a nice counterpart for this 'conceptually'
- End the sentence where now 'and' is, and put everything related to playback state in the next sentence